### ISSUE TITLE
Grid column resizing bug in RTL

### ISSUE THREAD

**Issue by idekkers** [telerik/kendo-angular#1074]
## I'm submitting a
* Bug report 

## Current behavior
column resizing in an RTL grid works the wrong way round, 
while moving the handle on the right of the cell, the left of the cell is resizing

## Expected behavior
while moving a handle, the same handle should be moving

## Minimal reproduction of the problem with instructions
http://plnkr.co/edit/zKhMCsrj0CVt6WOwYmQq?p=preview

## What is the motivation or use case for changing the behavior?
bug

## Environment

Package versions:
latest production kendo(1.6.1) and angular (5.0.1)

```

Browser:
- Chrome (desktop) version 62
- Firefox version 56

System:
- TypeScript version: 2.4.2
- Node version: 8.4.2
- Platform:   Windows

**Comment by rusev** [comment]
Thank you for reporting us the issue.

We will investigate the behavior further.

**Comment by rusev** [comment]
This issue is resolved in the `@dev` channel. The version of the builds is `v1.7.0-dev.201801031153`.

**Comment by ipeshev** [comment]
Fixed in v1.7.0-dev.201801050745
